ui/interface.py

```diff
@@ -70,14 +70,16 @@ def draw_box(arr, box):
 
 
 def make_ir_view(
-    arr, scores, boxes, landms, temps, calib_box, win_size, thr_min=32, thr_max=40
+    arr, scores, boxes, landms, temps, calibrate, calib_box, win_size, thr_min=32, thr_max=40
```

Comment:
May I suggest making default argument `calib_box=None` and later check for:
```python
if calib_box is not None:
    draw_box(arr, calib_box)
```
Then we accomplish the same logic with just 1 argument instead of 2.